changelog/60511.changed

```diff
@@ -0,0 +1 @@
+Changed "manufacture" grain to "manufacturer" for Solaris on SPARC to unify the name across all platforms. The old "manufacture" grain is now deprecated and will be removed in Sulfur
```

Comment:
Just one last change request. Can you change the filename to `60511.deprecated` since we are putting this grain on a deprecation path going forward. Thanks :)

Reply:
Sure, done :)